### DISCUSSION TITLE
Error: Command failed: depsync /bin/sh: depsync: command not found

### DISCUSSION THREAD

**Discussion by longjie2014** [Tencent/libpag#2120] [Q&A] [answered]
![image](https://github.com/Tencent/libpag/assets/7840917/c661cdb3-4919-46cb-bc9a-e7e4634f7288) 不知什么原因，在linux底下安装执行命令。

**Comment by kevingpqi123** [comment, marked as answer]
node 环境有问题，请严格按照 README 文档来操作

**Reply by gitchensai** [reply]
能说下是具体哪个步骤的问题吗，感谢
